src/.goreleaser.yaml

```diff
@@ -18,13 +18,15 @@ builds:
           - -X 'github.com/alcionai/corso/src/internal/events.RudderStackDataPlaneURL={{.Env.RUDDERSTACK_CORSO_DATA_PLANE_URL}}'
 
 archives:
-  - name_template: "{{ .ProjectName }}_{{ .Tag }}_{{ .Os }}_{{ .Arch }}"
-    replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  # this name template makes the OS and Arch compatible with the results of uname.
+  - name_template: >-
+      {{ .ProjectName }}_
+      {{- .Tag }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
```

Comment:
We should remove support for 32 bit archs in a future PR, and we should add arm64 support.

Reply:
That's a topic to take up with Georgi and Vaibhav.  I don't recall the original conversations around this, but I assume we retained it intentionally.  Maybe you can make a tech debt ticket for it?

Reply:
https://github.com/golang/go/wiki/GoArm